### PR TITLE
Short flag names

### PR DESCRIPTION
**Pull request by jcox250**
For #25 

* Used `pflag` package to add short flag names
* Added tests for short flag names
* Updated README with a list of commands

@davrodpin Would you like me to add the `pflag` package to the vendor directory? Or I could open an issue to add a dependency management tool like GoDep?

### CODE REVIEW

**Comment by davrodpin** [comment]
> @davrodpin Would you like me to add the `pflag` package to the vendor directory? Or I could open an issue to add a dependency management tool like GoDep?

Please add it to the `vendor` folder.

File: README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ INFO[0000] listening on local address                    local_address="127.0.0.
 ```
 
 ```sh
-$ mole -v -local 127.0.0.1:8080 -remote 172.17.0.100:80 -server user@example.com:22 -key ~/.ssh/id_rsa
+$ mole -V --local 127.0.0.1:8080 --remote 172.17.0.100:80 --erver user@example.com:22 --key ~/.ssh/id_rsa
```
Comment (davrodpin) [inline README.md]: typo
```suggestion
$ mole -V --local 127.0.0.1:8080 --remote 172.17.0.100:80 --server user@example.com:22 --key ~/.ssh/id_rsa
```

File: cli/cli.go [hunk 2]
```diff
@@ -36,23 +37,27 @@ func New(args []string) *App {
...
+	pf.VarP(&c.Remote, "remote", "r", "set remote endpoint address: [<host>]:<port>")
+	pf.VarP(&c.Server, "server", "s", "set remote endpoint address: [<host>]:<port>")
+	pf.StringVarP(&c.Key, "key", "k", "", "(optional) Set server authentication key file path")
+	pf.BoolVarP(&c.Verbose, "verbose", "V", false, "(optional) Increase log verbosity")
+	pf.BoolVarP(&c.Help, "help", "h", false, "list all options available")
+	pf.BoolVarP(&c.Version, "version", "v", false, "display the mole version")
+
+	pf.Parse(c.args[1:])
+
+	if len(c.args[1:]) == 0 {
```
Comment (davrodpin) [inline cli/cli.go]: The validation is already implemented on `Validate()`. Please remove it.

File: cli/cli.go [hunk 3]
```diff
@@ -144,6 +149,10 @@ func (h *HostInput) Set(value string) error {
 	return nil
 }
 
+func (h *HostInput) Type() string {
```
Comment (davrodpin) [inline cli/cli.go]: Please add documentation to this function. It would be good to say why this is needed.

**Comment by jcox250** [comment]
@davrodpin I was able to make some of the changes you requested. I'll be able to make the others tomorrow after work.

**Comment by jcox250** [comment]
@davrodpin I think I've made all of the updates you've requested. I could be wrong but I think the code coverage checker isn't happy because of the amount of untested code in `PrintUsage`.

**Comment by davrodpin** [comment]
Closing it for inactivity. #101 replaces it.

**Pull request closed without merging**